pkg/controller/certificatemanager/certificatemanager.go

```diff
@@ -365,3 +437,47 @@ func (cm *certificateManager) CreateTrustedBundle(certificates ...certificateman
 func (cm *certificateManager) CreateTrustedBundleWithSystemRootCertificates(certificates ...certificatemanagement.CertificateInterface) (certificatemanagement.TrustedBundle, error) {
 	return certificatemanagement.CreateTrustedBundleWithSystemRootCertificates(append([]certificatemanagement.CertificateInterface{cm.keyPair}, certificates...)...)
 }
+
+type annotationPassthru struct {
```

Comment:
Need to clean this struct up and comment it better - it's an implementation of `TrustedBundleRO` that can be used to load an existing bundle for read-only operations (which is what most controllers need). 

This split allows us to have one controller provisioning bundles, and another loading them in order to pass to render so that they can properly get annotations / volumes / etc without any chance of actually modifying the bundle.